### PR TITLE
exp implementation

### LINKED ISSUE #13158: exp

### PR DESCRIPTION
**Pull request by jatinS-dev**
closes #13158

### CODE REVIEW

File: ivy/functional/frontends/torch/tensor.py [hunk 1]
```diff
@@ -875,3 +875,7 @@ def __and__(self, other):
 
     def bitwise_xor(self, other, *, out=None):
         return torch_frontend.bitwise_xor(self._ivy_array, other)
+
+    @with_unsupported_dtypes({"1.11.0 and below": ("bfloat16",)}, "torch")
+    def exp(self, other, *, out=None):
```
Comment (juliagsy) [inline ivy/functional/frontends/torch/tensor.py:880]: exp needs only one argument, it takes `self` only, `other` may be removed

File: ivy/functional/frontends/torch/tensor.py [hunk 2]
```diff
@@ -875,3 +875,7 @@ def __and__(self, other):
...
+        return torch_frontend.exp(self._ivy_array, other)
```
Comment (juliagsy) [inline ivy/functional/frontends/torch/tensor.py:881]: same as above, `other` may be removed

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py [hunk 3]
```diff
@@ -5804,3 +5804,42 @@ def test_torch_instance_bitwise_xor(
...
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    frontend,
+    on_device,
+):
+    input_dtype, x = dtype_and_x
+    dtype = input_dtype[0]
+    if "int" in dtype:
+        x[1] = ivy.abs(x[1])
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:5830]: there'll only be one array generated now, so this does not apply now

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py [hunk 4]
```diff
@@ -5804,3 +5804,42 @@ def test_torch_instance_bitwise_xor(
...
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtype,
+        init_all_as_kwargs_np={
+            "data": x[0],
+        },
+        method_input_dtypes=input_dtype,
+        method_all_as_kwargs_np={
+            "exponent": x[1],
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:5838]: same as above, this is not needed

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey there! Thanks for contributing! There are some changes needed and there are left as comments below! Please request my review once you're done with the changes! Thanks a lot!

**Comment by jatinS-dev** [comment]
Greetings @juliagsy you can check now

File: ivy/functional/frontends/torch/tensor.py (hunk 1 above)
Comment (jatinS-dev) [reply ivy/functional/frontends/torch/tensor.py]: removed

File: ivy/functional/frontends/torch/tensor.py (hunk 2 above)
Comment (jatinS-dev) [reply ivy/functional/frontends/torch/tensor.py]: removed

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py (hunk 4 above)
Comment (jatinS-dev) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py]: done

**Review by juliagsy — APPROVED** [review]
Thanks for contributing! I'll merge in soon!

**Pull request merged**